theme_loftspace/static/src/scss/primary_variables.scss

```diff
@@ -121,18 +121,27 @@ $o-theme-font-configs: (
 // Website customizations
 //------------------------------------------------------------------------------
 
+// Change color preset for the footer
+$o-base-color-palette: map-merge($o-base-color-palette, (
+    'footer': 3,
+));
```

Comment:
o-base-color-palette should normally *never* be changed by a theme. Like its name advertises, it is meant to define the default values each potential key should have, meaning that a palette not defining a value for that key would use that default value. Doing this, you basically change all footer colors of all the palettes that do not define one and that the user can choose... very random.

If your theme wants to use the preset color n°3 for its footer, it should normally use a color palette which use that for its footer... so here I see you use "graphene-2" as color palette, the normal way would be to change the graphene-2 palette for all themes unfortunately... to see if that make sense.

Reply:
After working on it, this change is not needed and the theme can work with the default color of the footer 👍 .